core/client/fs/src/main/java/alluxio/client/file/cache/PageStore.java

```diff
@@ -170,27 +136,16 @@ ReadableByteChannel get(PageId pageId, int pageOffset) throws IOException,
   void delete(PageId pageId, long pageSize) throws IOException, PageNotFoundException;
 
   /**
-   * @return the number of pages stored
-   */
-  long pages();
-
-  /**
-   * @return the total size of pages stored in bytes
-   */
-  long bytes();
-
-  /**
-   * Gets all page ids.
+   * Restores the page store from a previous run.
    *
-   * @return collection of ids representing all pages loaded from disk
+   * @param initFunc function to apply during restore process
+   * @return true if successfully restored from previous state
    * @throws IOException if any error occurs
    */
-  Collection<PageInfo> getPages() throws IOException;
+  boolean restore(Predicate<PageInfo> initFunc) throws IOException;
```

Comment:
We need to indicate that this function must be called right after the constructor. The state of the page store is undetermined before this function is called.